spec/mailers/previews/ticket_preview.rb

```diff
@@ -0,0 +1,8 @@
+# Preview all emails at http://localhost:3000/rails/mailers/ticket
+class TicketPreview < ActionMailer::Preview
+  def new_ticket
+    user = User.last
+    ticket = Ticket.last
+    TicketMailer.new_ticket(user:, ticket_id: ticket.id)
```

Comment:
does it works when user and/or ticket are nil?

Reply:
nop. But it is just to check the preview email in DEV mode.  If I create a ticket in every call, the dev database will increase in size.